.ci/configure_containerd_for_kata.sh

```diff
@@ -15,6 +15,8 @@ readonly runc_path=$(command -v runc)
 sudo mkdir -p /etc/containerd/
 
 cat << EOT | sudo tee /etc/containerd/config.toml
+[debug]
+  level = "debug"
```

Comment:
This looks like a separate PR to me. It's unclear how this is related to the problem described in the PR title.

Reply:
yeah - I agree, adding DNM and WIP label